<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInformationClass</class>
 <widget class="QDialog" name="FileInformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileInformation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="itemGroup">
       <property name="title">
        <string>Item File</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Signature:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Items:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editItems">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Outfits:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editOutfits">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Effects:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="editEffects">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Projectiles:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="editProjectiles">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="HexDoubleSpinBox" name="editItemSignature">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>4294967296.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="spriteGroup">
       <property name="title">
        <string>Sprite File</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Signature:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="HexDoubleSpinBox" name="editSpriteSignature">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>4294967296.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Sprites:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editSprites">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pictureGroup">
       <property name="title">
        <string>Picture File</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Signature:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="HexDoubleSpinBox" name="editPictureSignature">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>4294967296.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Pictures:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editPictures">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HexDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>hexdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tibiaeditor.qrc"/>
 </resources>
 <connections/>
</ui>
